packages/spright-components/src/chat/input/index.ts

```diff
@@ -107,7 +107,10 @@ export class ChatInput extends FoundationElement {
 const sprightChatInput = ChatInput.compose({
```

Comment:
The [focusDelegates docs](https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#focus-delegation) have some discussion around tabindex behavior when using `focusDelegates: true`. Maybe we should include that as well?

Reply:
Think the docs should have an update to describe these manual test cases? I think the current wording of "will work properly" vs "won't" isn't clear enough to validate implementation.